docs/UserGuide.md

```diff
@@ -367,6 +379,20 @@ If your changes to the data file makes its format invalid, ReadyBakey will disca
 
 _Details coming soon ..._
 
+### Using Natural Dates
+
+In order to provide a quicker method of referring to near dates, ReadyBakey allows users to use natural dates to refer to the directly upcoming days of the week.
+The closest upcoming date that corresponds to the natural date's day input will be used. 
+
+This can be used in any command where dates are required, such as `incompleteo` and `addo`. 
+
+Format: `Day HH:MM`
```

Comment:
Might want to move all the natural date examples from addo to this section instead. I would also recommend to move this to the top, maybe before the introduction to all the examples?

Reply:
Agreed, I will reorganise the UG's commands once everyone is done with their edits so as to avoid unnecessary merge conflicts.

Reply:
Added the examples of incompleteo and addo into the examples for natural dates